### PR TITLE
feat: structure of domain entities

### PR DESCRIPTION
**Pull request by salet-gutierrez-jalafund**
## Trello Ticket

[ID-20](https://trello.com/c/GC2Bkxyz)

## What did I do?

I added the classes for the domain entities that are in accordance with the tables that are going to be created for the database, which each attribute will be required so that there is no problem when adding data to the database.

## Why did I do this?

The main requirements are that you have to have an Entities domain, which complies with the structure of the proposed UML.

## How did I do it?

I added the necessary classes with their necessary attributes with the help of the document provided for the creation of models for the tables, and also with the UML.

### CODE REVIEW

**Comment by salet-gutierrez-jalafund** [comment]
[ID-19: Project Initialization](https://trello.com/c/GC2Bkxyz/27-id-19-project-initialization)

File: src/ExpenseTrackerGrupo2/DataAccess/Entities/User.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.DataAccess.Entities;
+
+public class User
+{
+    private Guid _id;
+    private string _name;
```
Comment (alex-paca-jalafund) [inline src/ExpenseTrackerGrupo2/DataAccess/Entities/User.cs]: cant be a null name, its needed to be setted like 'required', there are more casses of this in the PR.

**Review by alex-paca-jalafund — CHANGES_REQUESTED** [review]

File: src/ExpenseTrackerGrupo2/DataAccess/Entities/User.cs (hunk 1 above)
Comment (salet-gutierrez-jalafund) [reply src/ExpenseTrackerGrupo2/DataAccess/Entities/User.cs]: below it is requested to be required, such as:

```
public required Guid Id
{
   get => _id;
   set => _id = value;
}
```

**Review by alex-paca-jalafund — APPROVED** [review]
Nice work.

**Pull request merged**